docs-next/pages/whats-new.mdx

```diff
@@ -0,0 +1,65 @@
+import { Markdown } from '../components/Page';
+
+# What's New in Keystone Next
+
+> Note: this page is WIP, we have a lot more things to add to it!
+
+In this major update, we've focused on improving Keystone's **interfaces**, including the way you
+configure and run Keystone projects, and a whole new Admin UI.
+
+We're also adding powerful new features to make Keystone the best headless content management system
+around, especially when you're using a component-based front end like React and Vue.
+
+#### 🌟 Here are some of the hilights:
+
+## Simpler Config with TypeScript
+
+We've massively simplified how you configure a Keystone project, and now have full TypeScript
+support so you get autocomplete and validation as you work.
+
+## All New Admin UI
+
+Our Admin UI has been completely rewritten with Next.js behind the scenes and a new Design System.
+It's a lot faster, and more customisable than ever before.
+
+Among other improvements, the Keystone Authoring UX is now aware of your application permissions, so
+users get the right field interfaces based on whether they can view or edit a field.
+
+## Document Field
+
+You can now set up a visual editor that manages components you have created - with custom props, and
+WYSIWYG previews. Perfect for authoring content that works with a Design System on the front-end.
+
+Check out the [Live Demo and Docs](./guides/document)
+
+export default ({ children }) => <Markdown>{children}</Markdown>;
+
+## Extensible GraphQL Schema
+
+Keystone now gives you an executable GraphQL Schema that you can extend with your own custom types,
+queries and mutations. It's perfect for adding your own logic into your app, or integrating with
+external services through a unified API.
+
+And the best part is, it's the GraphQL Standard, so you can integrate any other project that
+produces a GraphQL API with no additional work, and no lock-in!
+
+## Next.js-based Server
```

Comment:
```suggestion
## Next.js Server
```
"Next.js-based" is probably technically correct, but looks really clunky 🤔

Reply:
Agree, I'm worried that "Next.js Server" means something quite different though. Vote to leave it as-is for now, will think on how we could better phrase this later.